iceoryx_binding_c/test/moduletests/test_wait_set.cpp

```diff
@@ -142,37 +142,39 @@ TEST_F(iox_ws_test, SizeIsZeroWhenConstructed)
 
 TEST_F(iox_ws_test, SizeIsOneWhenOneUserTriggerIsAttached)
 {
-    EXPECT_EQ(iox_ws_attach_user_trigger_event(m_sut, m_userTrigger[0U], 0U, NULL),
+    EXPECT_EQ(iox_ws_attach_user_trigger_event(m_sut, m_userTrigger[0U], 0U, userTriggerCallback),
```

Comment:
Could you please also add a check/assert to the C binding to not silently fail when a nullptr is passed to this function?

Reply:
@dkroenke creating a separate issue for this would also be fine with me

Reply:
Let's deal with that in a separate PR: https://github.com/eclipse-iceoryx/iceoryx/pull/933